File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex6.c
#include <stdio.h>

main()
{
    //vars
    float salarioFixo, vendas, comissao, salarioFinal;

    printf("\t\t\tSalario Funcionario\n\n");

    //Entradas
    printf("Insira o salario fixo: ");
    scanf("%f",&salarioFixo);

    printf("Insira o valor das vendas: ");
    scanf("%f",&vendas);

    //Processamento
    comissao = vendas * 0.04;
    salarioFinal = salarioFixo + comissao;

    //Saída
    printf("Comissao: R$%.2f", comissao);
    printf("Salario Final: R$%.2f", salarioFinal);


}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex6.java
//Importando Scanner
import java.util.Scanner;

public class Ex6{	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars
	double salarioFixo, valorVendas, comissao, salarioTotal;
	
	System.out.println("Salário de um funcionário\n\n");
	
	//Entradas
	System.out.print("Insira o salário fixo: ");
	salarioFixo = input.nextDouble();
	
	System.out.print("Insira o valor das vendas: ");
	valorVendas = input.nextDouble();
	
	//Processamento
	comissao = valorVendas * 0.04;
	salarioTotal = salarioFixo + comissao;
	
	//Saída
	System.out.println("Comissão a receber: R$" + comissao);
	System.out.printf("Salário total a receber: R$ %.2f", salarioTotal);
		
	}
}